### PR TITLE
Add link to edit content

### PR DESCRIPTION
**Pull request by bkeepers**
what_is_open_source__ _github_guides

### CODE REVIEW

**Comment by mostafalarki1368** [comment]
> Thanks!

**Review by KelvinMatthew-shit — COMMENTED** [review]
readme.md@github

**Review by Jhonsshons — COMMENTED** [review]
Add

File: _config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 baseurl: "/open-source-handbook"
+branch: gh-pages
```
Comment (ratchanonsuttawas) [inline _config.yml:2]: +2 - +2

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply _config.yml:2]: +2 to +2

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply _config.yml:2]: 0cb9553a13e5fa90cd3a6423b36f42cd7540ab19

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply _config.yml:2]: +2 to + 2

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply _config.yml:2]: +2 to +2

**Review by ratchanonsuttawas — COMMENTED** [review]
+2 to +2 0cb9553a13e5fa90cd3a6423b36f42cd7540ab19

**Comment by ratchanonsuttawas** [comment]
Pull requests ca8d4e5dc0df7c5b975dc599ab6843720239d2cf

**Review by Frankbrids43 — COMMENTED** [review]
Law enforcement has been notified

**Pull request merged**